stable/ksoc-plugins/values.yaml

```diff
@@ -84,6 +84,71 @@ ksocRuntime:
     replicas: 3
     nodeSelector: {}
     tolerations: []
+  detectReachableVulnerabilities: false
+  runtimeProfileRules: |
+    - list: ksoc_k8s_namespaces_excluded_from_runtime_profiles
+      items:
+        - kube-system
+        - cert-manager
+        - calico-system
+        - calico-apiserver
+        - ksoc
```

Comment:
I guess we could still be running it for KSOC in our own clusters?

Reply:
It's possible to have Helm values file at installation time that overwrites or appends to the `ksoc_k8s_namespaces_excluded_from_runtime_profiles` list without modifying the actual logic of such rule. (That's a common pattern for the default / preinstalled rules as well.) Therefore, we just need agreement what's sensible default.

Reply:
That's tottally fine. Just thinking it would make a lot of sense to be able to dogfood our own software through this, therefore maybe a variable `excludedNamespaces` would work. This is not critical though, just a suggestion for future versions.

Reply:
I got your point. The `excludedNamespace` value would require moving the whole Falco customization (rules YAML) to the `ksoc-runtime-profile-configuration` ConfigMap's template, which @swade1987 suggested not to do.

For the initial rollout, I've removed the `ksoc` namespace from excluded list.

Let's get some feedback regarding configuration from early adopters. They may request allow list or label selectors, which is all legitimate use case. Last but not least, we are going to displace Falco so this configuration is subject to change anyway.

Reply:
As discussed, adding ksoc namespace back.